src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -174,8 +174,20 @@ internal ItemManager(RecentItemsManager recentItemsManager)
 
             // Use a monotonically increasing integer to keep track the original alphabetical order of each item.
             var currentIndex = 0;
-            var builder = ArrayBuilder<MatchResult>.GetInstance(data.InitialSortedList.Length);
 
+            var builder = ArrayBuilder<MatchResult>.GetInstance();
```

Comment:
💡 You can pass a length to `GetInstance`

Reply:
it just calls EnsureCapacity underneath 
http://sourceroslyn.io/#Microsoft.CodeAnalysis/ArrayBuilder.cs,355